doc/userdoc/contribute/templates_styleguides/styleguide.rst

```diff
@@ -0,0 +1,608 @@
+The NEST documentation style guide
+==================================
+
+.. contents:: On this page, you'll find
+   :local:
+
+Why do we have a style guide?
+-----------------------------
+
+This style guide was created to provide a single reference point for content
+creation in NEST. This helps ensure the NEST user-level documentation remains
+clear and consistent. The style choices we make here are meant to follow the
+major trends in technical writing for software.
+
+Contribute to the docs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to add or modify documentation, make sure to read through this guide before writing a contribution.
+
+We have templates for :doc:`Python example scripts <pyapi_template>` and the :doc:`PyNEST API docstrings <example_template>`.
+
+You will also need to know how to build the documentation locally on your machine: See
+:doc:`../../documentation_workflow/user_documentation_workflow`.
+
+For additional information, including the Git workflow and making a pull request see :doc:`../index`
+
+
+General guidelines about writing for NEST
+-----------------------------------------
+
+A few tips on writing documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Be concise: Use short words and sentences; avoid unnecessary modifiers
```

Comment:
This is not rendered as a list, just a sequence of paragraphs. Several items on the list miss the period at the end.

Reply:
All of these are rendering as lists for me.